run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.aerosol

```diff
@@ -2165,7 +2215,7 @@ ${RUNDIR_TES_CLIM_N2O}
 #==============================================================================
 # --- Annual scale factors ---
 #==============================================================================
-(((HTAP.or.XIAO_C3H8
+(((HTAPv3.or.HTAPv3_SHIP.or.XIAO_C3H8
```

Comment:
Do we still use these annual scale factors for HTAPv3 or can we remove the two HTAP options here leaving just `XIAO_C3H8`?

Reply:
I think we can remove the `_2008_2010`  scale factors since they were added for the original HTAP. In fact, we don't seem to be using any annual scale factors except for 6 and 7 (used for `XIAO_C3H8`. I would recommend removing those that aren't used avoid confusion.

Reply:
The SOAP entries are using scale factor 26, which is in that block of scale factors
```console
0 HTAPv3_SOAP_AGR $ROOT/HTAPv3/v2022-12/$YYYY/HTAPv3_CO_0.1x0.1_$YYYY.nc   CO_AGR  2000-2018/1-12/1/0 C xy kg/m2/s SOAP  26/280   1 4
```

Reply:
Or if you think that should be deleted I can take them out and just use scale factor 280

Reply:
Nevermind, 26 is outside of that scale factor  Very confusing.

Reply:
Now fixed in https://github.com/geoschem/geos-chem/commit/3f69125dbdf6ae180be14d943941683f4c7b9185
https://github.com/geoschem/geos-chem/blob/3f69125dbdf6ae180be14d943941683f4c7b9185/run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.aerosol#L2250-L2256

Reply:
@yantosca Could you also remove the obsolete scale factors from this section? That is, only scale IDs 6 and 7 should remain since those are the only annual scale factors used by XIAO_C3H8.

Reply:
Now removed in dd89da81003b7d9ebe96af4de90ecaf7620d290e